kubernetes/apps/prow.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: prow
+spec:
+  goTemplate: true
+  generators:
+    - clusters:
+        selector:
+          matchLabels:
+            clusterType: 'prow'
```

Comment:
I think we should have `prow-control-plane` and `prow-build` as Argo will be used for build clusters as well eventually. That way we avoid deploying stuff into wrong cluster by accident.

Reply:
Each cluster will have its own prow folder to deploy prow specific resources.

Reply:
Okay, I see, then it's all good.